contracts/BlankArt.sol

```diff
@@ -199,24 +224,34 @@ contract BlankArt is ERC721, EIP712, ERC721URIStorage {
         // make sure that the redeemer is paying enough to cover the buyer's cost
         require(msg.value >= (voucher.minPrice * amount), "Insufficient funds to redeem");
 
+        require(
+            amount <= voucher.tokenCount,
+            "Amount is more than the voucher allows"
+        );
+
+        // make sure voucher has not already been claimed. If true, it HAS been claimed
+        require(!voucherClaimed[_hash(voucher)], "Voucher has already been claimed");
```

Comment:
i just want to flag this for someone more knowledgable to doublecheck that this is the correct hash function to use? @fordN ?

Reply:
Thanks - agreed! I'd definitely appreciate any feedback from someone with more experience there.